javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,11 @@
+const dogYearOfBirth = 1985;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
```

Comment:
same with variable declaration

Reply:
instead of declaring the two deferent variables for dog year and humanyear you could use one like
const dogYear = dogYearFuture-dogYearOfBirth

Reply:
done. thank you.